packages/core/admin/admin/src/content-manager/pages/ListView/index.js

```diff
@@ -154,13 +154,15 @@ function ListView({
         });
       }
     },
+    // TODO: remove when we evaluate the need of the useCallback
+    // eslint-disable-next-line react-hooks/exhaustive-deps
     [formatMessage, getData, getDataSucceeded, notifyStatus, push, toggleNotification]
   );
 
   const handleConfirmDeleteAllData = useCallback(
     async (ids) => {
       try {
-        await axiosInstance.post(getRequestUrl(`collection-types/${slug}/actions/bulkDelete`), {
+        await fetchClient.post(getRequestUrl(`collection-types/${slug}/actions/bulkDelete`), {
```

Comment:
Why don't you use the object destructuring in the del method like in the other ones? It's nothing important, feel free to ignore it 😂 

const { post } = useFetchClient()

Reply:
I needed the fetchClient variable to execute the get method because we also imported the lodash get method